### PR TITLE
Upload metadata function for track2

### PR DESCRIPTION
**Pull request by 00Kai0**

### CODE REVIEW

File: tools/azure-sdk-tools/packaging_tools/auto_codegen.py [hunk 1]
```diff
@@ -37,6 +40,54 @@ def init_new_service(package_name, folder_name):
...
+
+    readme_file = str(Path(spec_folder, input_readme))
+    global_conf = config["meta"]
+    local_conf = config["projects"][readme_file]
+
+    cmd = ["autorest", input_readme]
+    cmd += build_autorest_options(global_conf, local_conf)
+
+    # metadata
+    _metadata = {
```
Comment (jsntcy) [inline tools/azure-sdk-tools/packaging_tools/auto_codegen.py:53]: use `metadata`

File: tools/azure-sdk-tools/packaging_tools/auto_codegen.py [hunk 2]
```diff
@@ -37,6 +40,57 @@ def init_new_service(package_name, folder_name):
...
+
+    metadata_file_path = os.path.join(package_folder, "_meta.json")
+    with open(metadata_file_path, "w") as writer:
+        json.dump(metadata, writer, indent=2)
+    _LOGGER.info(f"Saved metadata to {metadata_file_path}")
+
+    # Check whether MANIFEST.in includes _meta.json
+    require_meta = "include _meta.json\n"
+    manifest_file = os.path.join(package_folder, "MANIFEST.in")
+    if not os.path.exists(manifest_file):
```
Comment (jsntcy) [inline tools/azure-sdk-tools/packaging_tools/auto_codegen.py:78]: If MANIFEST.in doesn't exit, do we nee create one?

Comment (00Kai0) [reply tools/azure-sdk-tools/packaging_tools/auto_codegen.py:78]: I think we don't need to do it at this script. This file should be created from template. If not, it maybe something wrong when creating this package at first time.

**Review by jsntcy — APPROVED** [review]

**Pull request merged**